extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php

```diff
@@ -20,7 +20,6 @@ public function __construct() {
 	public function sendDigestToUser( $userID ) {
 
 		if ( $this->shouldNotSendDigest( $userID, $sendLogging = true ) ) {
-			$this->clearUserFromGlobalWatchlist( $userID );
```

Comment:
This call is actually redundant since we end up clearing the global watchlist right after we call `sendDigestToUser` [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/GlobalWatchlist/GlobalWatchlistTask.class.php#L15).